### PR TITLE
Bug 1216129 - Fix popups causing rocketbar to become collapsible. r=etienne

### PR DESCRIPTION
**Pull request by Cwiiis**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5425ee670b02974e7c7d0fcaf829fb5f65961d90

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9db544ab8449b8dc83e1baa01c928964a71b31c4

File: apps/system/js/app_chrome.js [hunk 1]
```diff
@@ -791,6 +791,12 @@
     };
 
   AppChrome.prototype.handleScrollAreaChanged = function(evt) {
+    // Make sure the scroll-area-changed is coming from the right element.
+    if (evt && (!this.app || !this.app.browser ||
```
Comment (etiennesegonzac) [inline apps/system/js/app_chrome.js]: this `!this.app` is probably overcautious since the `AppChrome` constructor expects it as a param.

Comment (Cwiiis) [inline apps/system/js/app_chrome.js]: That's what I thought, but I saw a null this.app while testing this with web activities, so... I dunno what's going on there but I'd rather keep it in :)

Comment (etiennesegonzac) [inline apps/system/js/app_chrome.js]: hum, we might want to file a bug for this then :) but yeah feel free to leave the check

**Pull request merged**